doc/governance/project-docs/GOVERNANCE.md

```diff
@@ -0,0 +1,59 @@
+
+**1.3 Lead Maintainer**. The Lead Maintainer is responsible for day-to-day management of the Project, including making releases if the Release Manager role is vacant. The Maintainers may elect or remove a Lead Maintainer through consensus or simply majority vote if consensus is not met. During their term, the Lead Maintainer should maintain a high level of involvement in the maintenance of the Project. If a Lead Maintainer is unresponsive at their listed handle for more than three months, the Maintainers may vote to remove the member.
+
+**1.4 Release Manager**. The Release Manager is a Maintainer with the temporary position of managing, curating milestones, and publishing a specific release. Unless otherwise delegated, the Release Manager prepares release notes and writes the blog post for the release, if needed. Due to the temporary and revolving nature of this role, it does not need to be specified in [MEMBERS.md](MEMBERS.md). Lead Maintainer or Project Owner may request a Maintainer to be Release Manager, or a Maintainer may volunteer for the role.
+
+**1.5 Contributors**. Contributors are those that have made contributions to the Project. Contributors can participate in the project by submitting, reviewing, and discussing GitHub Pull Requests and Issues and participating in open and public HoloViz Organization discussions on GitHub, Discourse, Slack, Gitter, Twitter, StackOverflow, or other communication channels. Contributors may also participate in live events. Contributors may not vote along with Maintainers, but can submit their input on a given topic to be considered during a consensus forming process. Anyone can be a Contributor - they can be affiliated with a legal entity or none.
+
+**1.6 Distinguished Contributors**. Membership in the Distinguished Contributors is meant to recognize the work of Contributors that have gone above-and-beyond in their work on the project, but it is not meant as a means of conveying power or responsibility. We are deliberately not setting arbitrary quantitative metrics (like “100 commits”) to avoid encouraging behavior that plays to the metrics rather than the Project’s overall well-being. The Maintainers will host an up to date list of Distinguished Contributors on the Project repository or website. The Maintainers may vote at any time on inducting new Distinguished Contributors.
+
+**1.7 Emeritus**. Emeritus members are former Maintainers that are no longer actively involved and can not vote. A Maintainer can become Emeritus by their own decision that has been expressed in writing to the other Project Maintainers, or if they become unresponsive at their listed handle for more than three months and the Maintainers have voted to remove the member as an active Maintainer. Emeritus members may return to active Maintainer status through a simple majority vote of Maintainers.
```

Comment:
wow that's a lot of roles.. Can one person be assigned to multiple roles?

Reply:
yes, one person can have multiple roles, where it makes sense